server/backend/testspg/crud/test_idim_proxy_service.py

```diff
@@ -22,25 +26,34 @@ class TestIdimProxyServiceClass(object):
     Testing IdimProxyService class with real remote API calls (TEST environment).
     """
 
-    search_params = IdimProxySearchParam(
+    # Valid test IDIR user.
+    search_params_idir = IdimProxySearchParam(
         **{"userId": "ianliu"}
-    )  # Valid test IDIR user.
+    )
+    # Valid test Business Bceid user
     search_params_business_bceid_same_org = IdimProxyBceidSearchParam(
         **{"searchUserBy": IdimSearchUserParamType.USER_ID,
-            "searchValue": TEST_VALID_BUSINESS_BCEID_USERNAME_ONE}
-    )  # Valid test Business Bceid user.
+            "searchValue": USER_NAME_BCEID_LOAD_3_TEST_CHILD_1}
+    )
+    # Valid test Business Bceid user.
     search_params_business_bceid_diff_org = IdimProxyBceidSearchParam(
         **{"searchUserBy": IdimSearchUserParamType.USER_ID,
-            "searchValue": TEST_VALID_BUSINESS_BCEID_USERNAME_TWO}
-    )  # Valid test Business Bceid user.
+            "searchValue": USER_NAME_BCEID_LOAD_2_TEST}
+    )
+    # IDIR test user_guid. Note, do not use it in other place.
+    # This is only for real integration test.
+    TEST_IDIR_USER_GUID = "E72A12C916A44A9581CF39E5DCDFFAE7"
```

Comment:
Hi @basilv , appreciate for your opinion in this case.
This test file is for real integration test for FAM connecting to IDIM-Proxy for IDIM search.
To do the real search, we need to use the **real** **USER_GUID** (for searching on BCeID user, based on IDIM requirement).
We previously already use **real** IDIR "**USER_NAME**" in other search IDIM search.
Do you feel it is ok here for Integration Test or should we have ticket to migrate these to Github secrets (to be feed on to the environment variables)?

Reply:
I'd feel more comfortable if this was a github secret.

Reply:
Thanks Basil. Same, I felt not comfortable as well initially. Will move it to github secret.